cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -174,17 +177,19 @@ func deleteInterface(networkContainerID string) error {
 		"/operation",
 		"DELETE"}
 
-	log.Printf("[Azure CNS] Going to delete network loopback adapter: %v", args)
 	c := exec.Command("cmd", args...)
 
 	loopbackOperationLock.Lock()
+	log.Printf("[Azure CNS] Going to delete network loopback adapter: %v", args)
 	bytes, err := c.Output()
 	loopbackOperationLock.Unlock()
 
 	if err == nil {
```

Comment:
do you think we can just combine this? a single line?

Reply:
Which lines are you suggesting to combine?